29_Arithmetic2Bench/app_resources/benchmarkSubgroup.comp.hlsl

```diff
@@ -0,0 +1,57 @@
+#pragma shader_stage(compute)
+
+#define operation_t nbl::hlsl::OPERATION
+
+#include "nbl/builtin/hlsl/glsl_compat/core.hlsl"
+#include "nbl/builtin/hlsl/glsl_compat/subgroup_basic.hlsl"
+#include "nbl/builtin/hlsl/subgroup2/arithmetic_portability.hlsl"
+
+#include "shaderCommon.hlsl"
+#include "nbl/builtin/hlsl/workgroup/basic.hlsl"
+
+typedef vector<uint32_t, ITEMS_PER_INVOCATION> type_t;
+
+uint32_t globalIndex()
+{
+    return nbl::hlsl::glsl::gl_WorkGroupID().x*WORKGROUP_SIZE+nbl::hlsl::workgroup::SubgroupContiguousIndex();
+}
+
+bool canStore() {return true;}
+
+template<class Binop, uint32_t N>
+static void subbench(NBL_CONST_REF_ARG(type_t) sourceVal)
+{
+    using config_t = nbl::hlsl::subgroup2::Configuration<SUBGROUP_SIZE_LOG2>;
+    using params_t = nbl::hlsl::subgroup2::ArithmeticParams<config_t, typename Binop::base_t, N, nbl::hlsl::jit::device_capabilities>;
```

Comment:
don't use JIT traits, you need to benchmark both (compile pipeline twice) see https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/190#discussion_r2036155341